db/migrate/20210810212018_add_xccdf_columns.rb

```diff
@@ -0,0 +1,69 @@
+class AddXccdfColumns < ActiveRecord::Migration[6.1]
+  def change
+    add_column :rules, :status, :string
+    add_column :rules, :status_justification, :text
+    add_column :rules, :artifact_description, :text
+    add_column :rules, :vendor_comments, :text
+    add_column :rules, :rule_id, :string, null: false
+    add_column :rules, :rule_severity, :string
+    add_column :rules, :rule_weight, :string
+    add_column :rules, :version, :string
+    add_column :rules, :title, :string
+    add_column :rules, :ident, :string
+    add_column :rules, :ident_system, :string, default: 'http://iase.disa.mil/cci'
+    add_column :rules, :fixtext, :text
+    add_column :rules, :fixtext_fixref, :string
+    add_column :rules, :fix_id, :string
+    remove_column :rules, :description, :string
+
+    create_table :checks do |t|
+      t.references :rule
+      t.string :system
+      t.string :content_ref_name
+      t.string :content_ref_href
+      t.text :content
+      t.timestamps
+    end
+
+    create_table :disa_rule_descriptions do |t|
+      t.references :rule
+      t.text :vuln_discussion
+      t.text :false_positives
+      t.text :false_negatives
+      t.boolean :documentable
+      t.text :mitigations
+      t.text :severity_override_guidance
+      t.text :potential_impacts
+      t.text :third_party_tools
+      t.text :mitigation_control
+      t.text :responsibility
+      t.text :ia_controls
+      t.timestamps
+    end
+
+    create_table :rule_descriptions do |t|
+      t.references :rule
+      t.text :description
+      t.timestamps
+    end
+
+    create_table :references do |t|
```

Comment:
Does this table need to have an association to rule/any other added table?

Reply:
It will eventually link with a rule (at least), but I'm not sure the best way to do it yet so I left it off. I'm not sure if it will be worthwhile to reuse references in the back-end between rules so we can find rules that all reference the same thing easier of if they are all sufficiently different/unique per rule. Need to look more into what the references are.